Documentation/ColumnsConfig/Type/Category/Index.rst

```diff
@@ -6,24 +6,14 @@
 TCA column type `category`
 ==========================
 
-..  versionadded:: 11.4
```

Comment:
You added here a hint for v11. Was this your intention? I thought this commit should only contain 12.x version hints?

Reply:
The red texts are the removed ones